src/bosh-google-cpi/action/create_vm.go

```diff
@@ -184,7 +185,14 @@ func (cv CreateVM) findZone(zoneName string, disks []DiskCID) (string, error) {
 	return "", fmt.Errorf("Could not find zone %q", zoneName)
 }
 
+func isGcpImageURL(s string) bool {
+	return strings.HasPrefix(s, "https://www.googleapis.com/compute/v1/projects/")
```

Comment:
@dgodd I see, I withdraw my objection then. LGTM, I'll leave it to @evandbrown to merge 👍